test/base/build.gradle.kts

```diff
@@ -20,16 +20,10 @@ import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTargetWithSimulatorTes
 plugins {
     id("org.jetbrains.kotlin.multiplatform")
     id("com.android.library")
-    id("io.realm.kotlin")
     id("realm-lint")
 }
 
-repositories {
-    google()
-    jcenter()
-    mavenCentral()
-    mavenLocal()
-}
+apply(plugin = "io.realm.kotlin")
```

Comment:
Can't you apply it in the `plugins` section even for local repositories when you have the plugin on the classpath with `classpath("io.realm.kotlin:gradle-plugin:${Realm.version}")` (as configured in `test/build.gradle.kts`?

Reply:
Yes, I remember running into issues in the plugin section and this fixed it, but I just tried moving it back and that also worked. So probably some other caching issue was at play.